instance/tests/api/test_openedx_appserver.py

```diff
@@ -145,7 +145,7 @@ def test_get_details(self):
         self.assertIn(('public_ip', None), server_data)
         # The API call will try to start the server, which will fail, since we're
         # not actually talking to an OpenStack instance when unit tests are running
-        self.assertIn(('status', 'failed'), server_data)
+        self.assertIn(('status', 'unknown'), server_data)
```

Comment:
@pomegranited Thanks for figuring that out and fixing it - we definitely should not have been making HTTP requests to example.com and depending on the return code :-/ Keeping it in this PR is totally fine.